website/docs/r/aws_auth_backend_identity_whitelist.html.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_auth_backend_identity_whitelist resource"
+sidebar_current: "docs-vault-resource-aws-auth-backend-identity-whitelist"
+description: |-
+  Configures the periodic tidying operation of the whitelisted identity entries.
+---
+
+# vault\_aws\_auth\_backend\_identity\_whitelist
+
+Configures the periodic tidying operation of the whitelisted identity entries.
+
+For more information, see the
+[Vault docs](https://www.vaultproject.io/api/auth/aws/index.html#configure-identity-whitelist-tidy-operation).
+
+## Example Usage
+
+```hcl
+resource "vault_auth_backend" "example" {
+  type = "aws"
+}
+
+resource "vault_aws_auth_backend_identity_whitelist" "example" {
+  backend       = "${vault_auth_backend.example.path}"
+  safety_buffer = 3600
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `backend` - (Optional) The path of the AWS backend being configured.
+
+* `safety_buffer` - (Optional) The amount of extra time, in minutes, that must
```

Comment:
Is `0` a valid value here, or is there a default?

Reply:
There's a default (72h: https://www.vaultproject.io/api/auth/aws/index.html#safety_buffer) that comes from Vault), but this can also be set to 0. Updating to GetOkExists so 0 can be used, thanks for pointing that out.